### PR TITLE
Milestone 8: Details modal

### PR DESCRIPTION
**Pull request by Dangelo-JAN**
## Creating popup modal in mobile version milestone 8: Details modal

In this milestone we:

- Update app.js
- Update index.html
- Update style.css

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -4,6 +4,63 @@ const close = document.querySelector('.close-icon');
...
+    </div>
+    <div class="c-container popup-buttons">
+        <button class="g-item8 pff-500 inner-margin-buttom" type="button"><a class="sff-500 a-card" href="${card.liveButton}">See Live<img class="inner-icon" src="img/live-icon.svg"></a></button>
+        <button class="g-item8 pff-500 inner-margin-buttom" type="button"><a class="sff-500 a-card" href="${card.sourceButton}">See Source<img class="inner-icon" src="img/github-icon.svg"></a></button>
+    </div>`;
+  backdrop.appendChild(modal);
+  const closePopup = document.querySelector('.close-popup');
+  closePopup.addEventListener('click', closeModal);
+}
+
```
Comment (gemmen29) [inline app.js]: Kindly make the modal scrollable so I can see the full design in the mobile version.
image

Comment (gemmen29) [inline app.js]: Kindly try to match the Figma design in the desktop version :
| Your design | Figma design |
| --- | --- |
| image | image |

- [x] Please increase the left and right spacing of the paragraph.
- [x] Kindly try to center the technologies list to match the Figma design.
- [x] Kindly make the `see live` and `see source` beside each other and increase its width to match the Figma design.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Dangelo-JAN, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Dangelo-JAN** [comment]
Following the improvements I maked the changes necessary for the template. We have to remember the user experience and the dimensions of the figma template desktop.

Figma Desktop
image

my version desktop
image

And the version mobile you can do the scroll

File: app.js (hunk 1 above)
Comment (Kingstalux) [reply app.js]: resolved 👍

Comment (Kingstalux) [reply app.js]: resolved

**Review by Kingstalux — APPROVED** [review]
Hi @Dangelo-JAN ,
# Status: Approved ✔️ ✔️ ✔️  
128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [x] (OPTIONAL) It would be great if you could make the desktop version of the pop up scrollable too so that it can adapt to all screen sizes 👍 
- Your site:
Annotation 2022-03-03 230718



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Dangelo-JAN** [comment]
About your comment, I thought exactly the same, but I don't want to do it that way because I don't want to have the problem of PR for UX again, but thanks I will make the improvement. Thanks a lot.

**Pull request merged**